docs/content/faq.md

```diff
@@ -58,3 +58,37 @@ and make sure that `protocolUseSSL` / `CMD_PROTOCOL_USESSL` is set to `true` if
 ## HedgeDoc fails executing migrations and does not start
 Unfortunately, older versions of HedgeDoc had some bugs regarding migrations and didn't always record that a migration was executed.  
 Have a look at the *[Troubleshooting Migrations](/guides/migration-troubleshooting/)* guide for more information.
+
+## Why does my interface look weird?
+
+If your interface looks like this
+
+![Broken HedgeDoc interface](./faq/interface_broken.png)
+
+instead of this
+
+![Not broken HedgeDoc interface](./faq/interface_okay.png)
+
+then you should check the following settings (or their corresponding environment variables if you e.g. use docker.):
+
+- `domain`
+- `protocolUseSSL`
+- `urlAddPort`
+
+Check [the configuration page](/configuration/) for an explanation.
+
+These settings control how HedgeCoc generates links to other pages and to assets, therefore you must set them from the outside point of view.
+
+Example: You access your HedgeDoc instance directly via `http://93.184.216.34:5812`. You must set:
```

Comment:
We should probably keep `3000` as the example port as this is the default port for HedgeDoc.